tests/test_resilientsession.py

```diff
@@ -53,3 +58,16 @@ def test_logging_with_connection_error(self):
     def tearDown(self):
         jira.resilientsession.logging.getLogger().removeHandler(self.loggingHandler)
         del self.loggingHandler
+
+
+@patch("requests.Session.get")
+def test_throttling_error_is_retried(mocked_method: Mock):
```

Comment:
we should patch `time.sleep()` to ensure that it is called too, so we actually retry with the exponential back off!

Otherwise test looks good, really appreciate it

Reply:
Good catch, it's actually not invoked right now as you pointed out in the other comment